features/api-resource-mgt/org.wso2.carbon.identity.api.resource.mgt.server.feature/resources/system-api-resource.xml

```diff
@@ -684,41 +684,41 @@
     <APIResource name="SCIM2 Bulk API" identifier="/scim2/Bulk" requiresAuthorization="true"
                  description="API representation of the SCIM2 Bulk API" type="TENANT_ADMIN">
         <Scopes>
-            <Scope displayName="Create Bulk Resource" name="internal_bulk_mgt_create" 
+            <Scope displayName="Create Bulk Resource" name="internal_bulk_resource_create"
                     description="Create new SCIM resources in bulk in the organization (root)"/>
-            <Scope displayName="View Bulk Resource" name="internal_bulk_mgt_view" 
+            <Scope displayName="View Bulk Resource" name="internal_bulk_resource_view"
```

Comment:
Lets remove the all bulk resource mgt scopes except the create one since we only have the POST call